docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc

```diff
@@ -0,0 +1,64 @@
+= Generic Pre-Install Checklist
+
+[abstract]
+To successfully install an OpenShift 4 cluster some pre-requisites are needed and some questions need to be answered before starting the installation. This page helps to be prepared for the cluster planning and installation.
+
+== Checklist
+
+* [ ] <<Naming and DNS>> defined?
+* [ ] <<Resource requirements>> defined?
+* [ ] <<Infrastructure>> defined?
+* [ ] <<Network requirements>> defined?
+
+== Naming and DNS
+
+DNS names are very important, changing them later is usually not possible or a huge amount of (error prone) work. Plan the following points carefully:
+
+* Base domain
+
+Also have a look at xref:explanation/dns_scheme.adoc[DNS Scheme] to get an idea how DNS records are handled.
+
+== Resource requirements
+
+The main purpose of the cluster will be to run the users applications and these applications have specific resource requirements.
+
+This needs to be taken into consideration when answering the following questions:
+
+* What kind of virtual machines will be needed to satisfy the resource requirements (vCPU / Memory) for running the users workload?
+* How many user application Pods will be scheduled (just an estimate)?
+* How many worker nodes will be needed?
+* How much room for failure and maintenance (`n+1` redundancy) is needed?
+
+Don't go below the defined minimum requirements.
+
+== Infrastructure
+
+Collect the following information about the infrastructure:
+
+* What infrastructure will be used (which cloud)?
+* In which account will the cluster be installed? Gather the needed access credentials with sufficient rights.
+
+Depending on what the infrastructure supports:
+
+* Which region and zone will be used for the cluster?
+* Do the worker nodes have to be spread over multiple availability zones (AZ)?
+
+== Network requirements
+
+Depending on the requirements to the network, the following information need to be prepared:
+
+* What's the IP subnet for the nodes?
+* Which IP network can be used for `clusterNetwork` (SDN)? Should be a `/14`.
```

Comment:
I'd suggest to turn these questions: "Is a change to the default network necessary?"

Reply:
Rephrased, please check again.